### PR TITLE
Update RN docs + add terms

### PR DESCRIPTION
**Pull request by mironiasty**
## Description

Quick change to fix RN documentation (`connect` vs `joinRoom`). And also add simple terms page

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,41 +1,41 @@
...
-```
-$ USE_SSH=true yarn deploy
-```
-
-Not using SSH:
-
-```
-$ GIT_USER=<Your GitHub username> yarn deploy
-```
+### Docosaurus
```
Comment (czerwiukk) [inline README.md:39]: **typo:** Docusaurus

File: guide/setup/index.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+Single video or audio streamed by paritcipant. It could be camera view, shared screen or microphone audio. Each
+participant could stream few tracks.
+
+### Server token
+
+Secret token that should be stored on your backend. It allows to create rooms and add participants.
+
+### Participant token
+
+Token that your backend shoudl pass to end client as authentication method for user to access room.
```
Comment (czerwiukk) [inline guide/setup/index.md:28]: **typo:** should

File: guide/react-native/connecting.mdx [hunk 3]
```diff
@@ -28,14 +28,20 @@ const { url, token } = await response.json();
...
 
-In order to connect, you just need to call `connect` method with data from previous step:
+In order to connect, you just need to call `joinRoom` method with data from previous step:
 
 ```ts
-import { connect } from "@fishjam-cloud/react-native-client";
+import { joinRoom } from "@fishjam-cloud/react-native-client";
 
-await connect(url, token);
+await joinRoom(url, token);
```
Comment (MiloszFilimowski) [inline guide/react-native/connecting.mdx:36]: **issue (blocking):** joinRoom also need third parameter of ParticipantMetadata.

Comment (mironiasty) [reply guide/react-native/connecting.mdx:36]: Fixed with this PR: https://github.com/fishjam-cloud/mobile-client-sdk/pull/129

**Review by MiloszFilimowski — APPROVED** [review]

File: guide/react-native/connecting.mdx (hunk 3 above)
Comment (czerwiukk) [inline guide/react-native/connecting.mdx:36]: **polish:** 
```suggestion
await joinRoom(fishjamUrl, participantToken);
```

Comment (czerwiukk) [reply guide/react-native/connecting.mdx:36]: 🇵🇱

**Pull request merged**